base_app/base.py

```diff
@@ -18,53 +11,38 @@ def home():
 
 @app.route('/api/v1/recommendations', methods=['GET'])
 def index():
-    YELP_API_KEY = os.getenv('YELP_API_KEY')
-    headers = {'authorization': 'Bearer ' + YELP_API_KEY}
-    lat = request.args['latitude']
-    long = request.args['longitude']
-    url = 'https://api.yelp.com/v3/businesses/search'
-    if 'price' in request.args.keys():
-        price = request.args['price']
-        params = {'latitude': f'{lat}', 'longitude': f'{long}', 'price': f'{price}'}
-    elif 'categories' in request.args.keys():
-        cuisine = request.args['categories']
-        params = {'latitude': f'{lat}', 'longitude': f'{long}', 'term': f'{cuisine}'}
-    response = requests.get(url, params=params, headers=headers)
-    json_data = json.dumps(response.json())
-    restaurants = Restaurant.from_json(json_data)
-    restaurant = random.choice(restaurants)
-    schema = RestaurantSchema()
-    result = schema.dump(restaurant)
+    service = YelpService()
+    params = format_params(request.args)
+    return service.get_recommendation(params)
 
-    return result
 
 @app.route('/api/v1/photos', methods=['GET'])
-def get_photos():
-    YELP_API_KEY = os.getenv('YELP_API_KEY')
-    headers = {'authorization': 'Bearer ' + YELP_API_KEY}
-    lat = request.args['latitude']
-    long = request.args['longitude']
-    cuisine = request.args['categories']
-    params = {'latitude': f'{lat}', 'longitude': f'{long}', 'term': f'{cuisine}'}
-    url = 'https://api.yelp.com/v3/businesses/search'
+def show():
+    service = YelpService()
```

Comment:
I think we need to combine this with the /api/v1/recommendations endpoint. My notes say we talked about pulling one restaurant and then using its ID to call the endpoint that has the photos (so there would effectively be two api calls back to Yelp before returning JSON to the FE).

Reply:
We can do that Sunday

Reply:
We also will need to refactor PR #38 Random restaurant endpoint to have it use the Service.